VultisigApp/VultisigApp/View Models/JoinKeysignViewModel.swift

```diff
@@ -273,4 +273,23 @@ class JoinKeysignViewModel: ObservableObject {
             chain: payload.coin.chain
         )
     }
+    
+    func blowfishSolanaTransactionScan() async throws -> BlowfishResponse? {
```

Comment:
throwing function should not return optional value

Reply:
Removed the throws then